public/index.html

```diff
@@ -0,0 +1,82 @@
+<!-- public/index.html -->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<title>Chat App</title>
+<style>
+  body {
+    background-color: black;
+    color: white;
+    font-family: Arial, sans-serif;
+  }
+  #chat {
+    max-width: 600px;
+    margin: 20px auto;
+    padding: 10px;
+    border: 1px solid #333;
+  }
+  #messages {
+    height: 300px;
+    overflow-y: auto;
+    margin-bottom: 10px;
+    padding: 5px;
+    background: #111;
+  }
+  #messageInput {
+    width: calc(100% - 60px);
+    margin-right: 10px;
+    background-color: dimgray;
+    color:white;
+  }
+  button {
+    width: 50px;
+    background-color: dimgray;
+    color:white;
+  }
+</style>
+</head>
+<body>
+<h1> NodeJS Cortext sample app </h1>
+<div id="chat">
+  <div id="messages"></div>
+  <input type="text" id="messageInput">
+  <button onclick="sendMessage()">Send</button>
```

Comment:
**Security Issue**: The 'sendMessage' function is directly invoked from an inline event handler, which can lead to Cross-Site Scripting (XSS) vulnerabilities if user inputs are not properly sanitized before being used in the DOM. <br> **Fix**: Remove the inline event handler and attach event listeners securely using JavaScript. This prevents potential XSS by ensuring that event handling is done in a way that does not expose the application to injection of malicious scripts. <br> **Code Suggestion**: 
 ```
 -  <button onclick="sendMessage()">Send</button>
 +  <button id="sendButton">Send</button>
 +  <script>
 +    document.getElementById('sendButton').addEventListener('click', sendMessage);
 +  </script>
 ```